sklearn/ensemble/_gb.py

```diff
@@ -972,19 +984,23 @@ class GradientBoostingClassifier(ClassifierMixin, BaseGradientBoosting):
     GB builds an additive model in a
     forward stage-wise fashion; it allows for the optimization of
     arbitrary differentiable loss functions. In each stage ``n_classes_``
-    regression trees are fit on the negative gradient of the
-    binomial or multinomial deviance loss function. Binary classification
+    regression trees are fit on the negative gradient of the loss function,
+    e.g. binary or multiclass log loss. Binary classification
     is a special case where only a single regression tree is induced.
 
     Read more in the :ref:`User Guide <gradient_boosting>`.
 
     Parameters
     ----------
-    loss : {'deviance', 'exponential'}, default='deviance'
-        The loss function to be optimized. 'deviance' refers to
-        deviance (= logistic regression) for classification
-        with probabilistic outputs. For loss 'exponential' gradient
-        boosting recovers the AdaBoost algorithm.
+    loss : {'log_loss', 'exponential'}, default='log_loss'
```

Comment:
I have seen both. E.g. the deprecation of the "mae" and "mse" criterion in trees are also not listed anymore during the deprecation cycle. They are only listed under deprecated.
I personally prefer it this way, but will do as reviewers wish.

Reply:
There's also the default value that is directly changed. The thing is that it's just a renaming that gives the same model, so maybe it's acceptable. I'm not sure if it can be a problem ?

Reply:
> I personally prefer it this way, but will do as reviewers wish.

I don't have a strong opinion, but I'd like that we always do it the same way (I didn't remember that we did it this way in the trees). Maybe we should explain haw to update the doc in the contributing guide.

Reply:
#19310 kept "mae" and "mse", but #21314 removed them before the end of the cycle.
>  don't have a strong opinion

Me neither. If I have a wish, then to get over it fast :wink:

Reply:
So I re-introduce them.